draft-ietf-tls-dtls13.md

```diff
@@ -1130,12 +1132,14 @@ fragmentation, DTLS modifies the TLS 1.3 handshake header:
             case client_hello:          ClientHello;
             case server_hello:          ServerHello;
             case end_of_early_data:     EndOfEarlyData;
+            case new_session_ticket:    NewSessionTicket;
```

Comment:
I see NST (4) as before EOED (5) in the registry.

Reply:
Were we planning to order these by codepoint value in the final RFC?

Reply:
Yes ... will submit a new PR now to reorder